careers/students-and-graduates/index.html

```diff
@@ -0,0 +1,221 @@
+{% extends "layout-side-nav.html" %}
+{% import "_vars-careers.html" as vars with context %}
+{%- set active_nav_id = 'students-and-graduates' -%}
+
+{% block title -%}
+    Students and Graduates
+{%- endblock %}
+
+{% block desc -%}
+
+{%- endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom
+{%- endblock %}
+
+{% block content_main %}
+
+    <section class="media block block__flush-top block__sub">
+
+            <h1>Students and graduates</h1>
+            <p class="h3">
+                The CFPB is committed to creating and sustaining
+                a new generation of leaders.
+                There are many opportunities for you to make an immediate impact.
+            </p>
+    </section>
+
+    <section class="students-and-graduates">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/8ae2c7950368992324794c7aa89771476ae9dabd and https://github.com/cfpb/cfgov-refresh/commit/4d932d3f1245986ac6ecbdd103eb64f18013ed66